src/view.rs

```diff
@@ -402,6 +405,10 @@ impl View {
         self.ops.push(ViewOp::Yank(self.active_layer_id, area));
     }
 
+	pub fn flip(&mut self, area: Rect<i32>, dir: Flip) {
+		self.ops.push(ViewOp::Flip(self.active_layer_id, area, dir));
+	}
```

Comment:
Indentation is messed up in a few places, make sure to run `cargo fmt` in the root of the project

Reply:
Oh, welp, I hadn't noticed there was `.rustfmt.toml` in the repository, so I was afraid I might break things with `cargo fmt` :sweat_smile: I'll fix it right away

Reply:
formatting fixed